app/controllers/api/v1/playlists_controller.rb

```diff
@@ -0,0 +1,69 @@
+module Api 
+  module V1
```

Comment:
I found this syntax in the tutorial article : `module Api::V1`
[The article](https://medium.com/swlh/a-deeper-dive-into-api-versioning-938b0cb58765)
What do you think?

Reply:
Seems the current nested syntax can avoid in some case future potential conflits. @Yous91 is looking throug